docs/src/main/asciidoc/stork-registration.adoc

```diff
@@ -0,0 +1,171 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Automatic Service Registration with SmallRye Stork for Quarkus
```

Comment:
You shouldn't put your titles all caps. It's agains The Rules (tm). Same for all the other titles.